src/main/app/case/case.ts

```diff
@@ -327,6 +327,8 @@ export interface Case {
   miamStart?: string;
   miamWillingness?: string;
   miamNotWillingExplnation?: string;
+  c1A_supervisionAgreementDetails?: string;
```

Comment:
@monalisasarkar1 , as discussed in today's call, please remove them since i believe there is no conditional routing/mapping for these fields